server/src/main/java/org/hyperskill/community/flashcards/card/ExampleDataInitializer.java

```diff
@@ -37,7 +37,7 @@ public class ExampleDataInitializer {
     private static final String categoryCollection = "category";
     private static final String userJsonPath = "/json/users.json";
     private static final String flashcardsJsonPath = "/json/flashcards.json";
-    private final ObjectMapper objectMapper;
+    private final ObjectMapper objectMapper = new ObjectMapper();
     private final UserMapper userMapper;
     private final MongoTemplate mongoTemplate;
```

Comment:
why not have it injected as the bean that is there anyway? It has no state..

Reply:
If it's injected from the context, I'll need to mock it in unit tests. Not a big deal, of course, but why?